code/game/objects/items/theft_tools.dm

```diff
@@ -263,3 +263,24 @@
 	playsound(src, 'sound/effects/supermatter.ogg', 50, TRUE)
 	QDEL_NULL(sliver)
 	update_icon()
+
+//Don't know where else to put fancy syndicate tools
+
+/obj/item/wrench/syndie
+	name = "suspicious-looking wrench"
+	desc = "It’s one of those fancy wrenches that you turn backward without twisting the bolt for faster action."
+	icon_state = "wrench_syndie"
+	toolspeed = 0.5
+
+/obj/item/crowbar/syndie
```

Comment:
this crowbar has less force than the red crowbar which was in the toolbox previously

Reply:
that's very unlikely to come up

Reply:
This is a black and RED crowbar (and a cool/advanced syndicate one at that), so we should give it at least the same force as a normal red crowbar.

Reply:
You're bikeshedding